package.json

```diff
@@ -36,20 +34,24 @@
     "react-dom": ">=16"
   },
   "devDependencies": {
-    "@babel/cli": "^7.19.3",
-    "@babel/core": "^7.20.5",
-    "@babel/plugin-transform-modules-commonjs": "^7.19.6",
-    "@babel/preset-react": "^7.18.6",
-    "@storybook/addon-essentials": "^6.5.14",
-    "@storybook/addon-links": "^6.5.14",
-    "@storybook/addon-storysource": "^6.5.14",
-    "@storybook/react": "^6.5.14",
+    "@storybook/addon-essentials": "^7.0.4",
+    "@storybook/addon-links": "^7.0.4",
+    "@storybook/addon-storysource": "^7.0.4",
+    "@storybook/react": "^7.0.4",
+    "@storybook/react-vite": "^7.0.4",
+    "@types/react": "^17.0.2",
+    "@types/react-dom": "^17.0.2",
+    "@vitejs/plugin-react": "^3.1.0",
     "cross-env": "^7.0.3",
     "eslint": "^8.29.0",
     "eslint-config-zakodium": "7.0.0",
+    "eslint-plugin-storybook": "^0.6.11",
```

Comment:
Is this used even if we don't extend it in the eslint config?

Reply:
It has been auto-added from the storybook migrator.
Now it's added in eslint config of project.